java/com/google/copybara/buildozer/testing/BUILD

```diff
@@ -19,15 +19,15 @@ licenses(["notice"])
 genrule(
     name = "normalised_buildozer",
     testonly = 1,
-    srcs = ["@buildtools//buildozer"],
```

Comment:
buildtools isn't in the bcr yet, this repo vendors the downloads directly from that repo's releases

Reply:
Adding links regarding buildtools migration to bzlmod:
- https://github.com/bazelbuild/buildtools/issues/1204
- https://github.com/bazelbuild/buildtools/pull/1245

Reply:
We currently automatically vendor buildtools from HEAD because we had needs to be pretty close to the edge. I wonder if this can be treated like io_bazel?

Reply:
the problem is that pulling in buildtools that way would require pulling rules_go and resolving the go.mod transitive dependencies of that repo as well.

Reply:
I see. I will look into merging this provided we don't end up with too much version skew for build_tools vs the internal monorepo.

Reply:
sounds good, i don't feel like there are a ton of changes there enough to cause major drift